cpp/farm_ng/core/enum/CMakeLists.txt

```diff
@@ -21,15 +9,25 @@ farm_ng_add_library(farm_ng_core_enum
 
 target_link_libraries(farm_ng_core_enum INTERFACE farm_ng_core::farm_pp)
 
-set(FARM_NG_ENUM_TESTS
+file(RELATIVE_PATH include_path_abs ${CMAKE_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+
+# TODO: Use Filesets?
+target_include_directories(farm_ng_core_enum INTERFACE
+"$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/${include_path_abs}>"
+"$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
+
+
+set(TEST_BASENAMES
       enum
       enum_without_iostream
       enum_flags
       enum_flags_without_ostream)
 
-foreach(test_basename ${FARM_NG_ENUM_TESTS})
-    farm_ng_add_test(${test_basename}
-                      PARENT_LIBRARY farm_ng_core_enum
-                      LINK_LIBRARIES farm_ng_core_enum
-                      LABELS small)
+foreach(test_basename ${TEST_BASENAMES})
+  set(test_target ${test_basename}_test_farm_ng_core_enum)
+  set(test_src ${test_basename}_test.cpp)
+  add_executable(${test_target} ${test_src})
+  target_link_libraries(${test_target} farm_ng_core_enum GTest::gtest_main)
+  add_test(NAME ${test_target} COMMAND ${test_target})
+  set_property(TEST ${test_target} PROPERTY LABELS small)
 endforeach()
```

Comment:
Changes in these CMakeLists.txt files are just experimenting with how farm_ng helper macros might be updated/refactored